src/domain/models/Dump.ts

```diff
@@ -4,6 +4,6 @@ export interface Dump {
     readonly id: Snowflake;
     readonly content?: string;
     readonly embeds?: object[];
-    readonly channelid?: Snowflake;
+    readonly channelid: Snowflake;
```

Comment:
Dumps only live for ~1 week, and the code since the typescript release has always set a channel on every insert to the table, so by now its definitely the case that they all have channel ids!